src/lib/consensus/proof_of_signature.ml

```diff
@@ -12,8 +12,8 @@ module Global_public_key = struct
   global_signer_real]
 
   let compressed =
-    Public_key.Compressed.of_base64_exn
-      "KBWuaAm5Sl5jH/dlpiTKQeUUsty/4Rq6Xz2Py2Y2i/VweJmDHwUAAAAB"
+    Snark_params.Tick.Inner_curve.one
+    |> Non_zero_curve_point.of_inner_curve_exn |> Public_key.compress
```

Comment:
This affects our secret gobal public key in the keybase repo too. Can you update that as required?

Reply:
Oh I thought we weren't using that anymore?

Reply:
Can you help document the technique required to seed the keys with this code change?